draft-ietf-acme-acme.md

```diff
@@ -765,6 +783,12 @@ notAfter (optional, string):
 : The requested value of the notAfter field in the certificate, in the date
 format defined in {{!RFC3339}}.
 
+error (optional, object):
+: The error that occurred while the server was issuing the certificate, if any.
```

Comment:
I don't really like this "while".

And the Note at the end of this paragraph should hint at the problem.

Instead:
"An error that occurred after authorizations were validated but before the server was able to issue a certificate (including its attempt to issue the certificate)." or
"An error that occurred after authorizations were validated but that prevented the server from issuing the certificate."

This obviates the need for the note.